platform/options.api/src/org/netbeans/modules/options/OptionsPanel.java

```diff
@@ -394,7 +393,7 @@ public void focusLost(FocusEvent e) {
         showHint(true);
         
         pCategories = new JPanel (new BorderLayout ());
-        pCategories.setBorder (BorderFactory.createMatteBorder(0,0,1,0,isFlatLaf ? UIManager.getColor("Separator.foreground"): Color.lightGray)); //NOI18N
+        pCategories.setBorder (BorderFactory.createMatteBorder(0,0,1,0,getUIColorOrDefault("nb.options.categories.separatorColor", Color.lightGray))); //NOI18N
```

Comment:
This changes semantics - should this be three level fallback?

- if `nb.options.categories.separatorColor` is defined use that
- if `Separator.foreground` is defined use that
- else fall back to default

Reply:
This looks like the same semantics to me. And very glad to see this change.

Reply:
I see a changed behavior here - if that is intend to change, I'm good with it.

Reply:
@matthiasblaesing unless you're proposing to keep the flatlaf check in there, what you suggested would return separator.foreground for all look and feels wouldn't it?

Reply:
@neilcsmith-net sorry, I overlooked the isFlatLaf check.